css-highlight-api-1/Overview.bs

```diff
@@ -450,6 +461,26 @@ Priority of Overlapping Highlights</h4>
 		</div>
 	</div>
 
+<h4 id=highlight-types>
+Highlight types</h4>
+
+	A [=custom highlight=]'s {{Highlight/type}} attribute is used by authors to specify the semantic
+	meaning of the highlight. This allows assistive technologies to include this meaning when
```

Comment:
I think it is important to explain here why we have chosen this initial set of types. That can also tie into the note on line 481.